openmetadata-ui/src/main/resources/ui/cypress/support/commands.js

```diff
@@ -79,3 +82,27 @@ Cypress.on('uncaught:exception', (err) => {
     return false;
   }
 });
+
+Cypress.Commands.add('storeSession', (username, password) => {
+  cy.session([username, password], () => {
+    cy.visit('/');
+    cy.get('[id="email"]').should('be.visible').clear().type(username);
+    cy.get('[id="password"]').should('be.visible').clear().type(password);
+    interceptURL('POST', '/api/v1/users/login', 'login');
+    interceptURL('GET', '/api/v1/users/*', 'loadPage');
+    cy.get('[data-testid="login"]')
+      .contains('Login')
+      .should('be.visible')
+      .click();
+    verifyResponseStatusCode('@login', 200);
+    verifyResponseStatusCode('@loadPage', 200);
+    cy.url().should('eq', `${BASE_URL}/my-data`);
+    cy.get('[data-testid="tables"]').should('be.visible');
+  });
+});
+
+Cypress.Commands.add('login', () => {
```

Comment:
It's the same as we are doing login we are not using cookies at all

Reply:
As we discussed it restores the cookies, and works as expected, I have added comments for the same